MementoMori/MementoMori.Server/Service/DeckHelper.cs

```diff
@@ -126,5 +126,33 @@ public async Task DeleteDeckAsync(Guid deckId, Guid requesterId)
                 throw new KeyNotFoundException();
             }
         }
+        public async Task<UserDecksDTO[]?> getUserDecks(Guid userId)
+        {
+            var userDecks = await _context.Decks
+                .Where(deck => deck.CreatorId == userId)
+                .Select(deck => new UserDecksDTO
+                {
+                    Id = deck.Id,
+                    Title = deck.Title
+                })
+                .ToArrayAsync();
+            return userDecks;
+        }
+
+        public async Task<bool> HasAccessToDeck(Guid userId, Guid deckId)
```

Comment:
Is this even needed? As the function that gets the deck must check it anyway.
Or is it used for something else? If so entire deck should be passed instead of filtering for it again